.gitignore

```diff
@@ -105,3 +105,8 @@ target/
 !config_examples/config_ftx.example.json
 !config_examples/config_full.example.json
 !config_examples/config_kraken.example.json
+doc/doc-filelist.js
+.gitignore
+.gitignore
+doc/doc-style.css
+doc/doc-script.js
```

Comment:
I don't think these files make sense to include in a gitignore file.

they're not part of the repository - and excluding .gitignore is kinda dangerous.